model-engine/model_engine_server/core/loggers.py

```diff
@@ -123,10 +119,14 @@ def make_json_logger(name: str, log_level: int = logging.INFO) -> logging.Logger
     in_kubernetes = os.getenv("KUBERNETES_SERVICE_HOST")
     if in_kubernetes:
         stream_handler.setFormatter(CustomJSONFormatter())
+    else:
+        stream_handler.setFormatter(CustomJSONFormatter())
```

Comment:
If we're doing this in both branches, can we just get rid of the `if` and `else`?

Reply:
I actually think we should keep the if/else logic and use a basic logging config for local tests ( generally more readable, just had to have the JSON enriched version for testing the trace id / request id changes )